packages/components/src/Badge/Badge.component.js

```diff
@@ -65,16 +61,15 @@ function Badge({
 	category,
 	className,
 	children,
-	disabled,
+	disabled = false,
 	display = SIZES.large,
 	icon,
 	id,
 	label,
 	onDelete,
 	onSelect,
-	selected,
+	selected = false,
 	style,
-	t,
```

Comment:
t is not used, just drilled down. Let's use it directly in component that needs it.